store/sqlstore/audit_store.go

```diff
@@ -58,26 +58,26 @@ func (s SqlAuditStore) Get(user_id string, offset int, limit int) (model.Audits,
 		Limit(uint64(limit)).
 		Offset(uint64(offset))
 
-	if len(user_id) != 0 {
-		query = query.Where(sq.Eq{"UserId": user_id})
+	if len(userId) != 0 {
+		query = query.Where(sq.Eq{"UserId": userId})
 	}
 
 	queryString, args, err := query.ToSql()
 	if err != nil {
-		return nil, model.NewAppError("SqlAuditStore.Get", "store.sql_audit.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrapf(err, "failed to create query to get Audit with userId=%s", userId)
```

Comment:
I did a check for errors from `ToSql` and we are not returning the actual value with the error. I would like to be consistent here too.

Here are the errors for ToSql:
```
channel_store.go-		return nil, errors.Wrap(err, "failed to create query")
channel_store.go-		return 0, errors.Wrap(err, "failed to create query")
channel_store.go-		return nil, errors.Wrap(err, "channel_members_tosql")
```

Maybe use `audits_tosql`? It's fine to keep the others like that as long as we later make a full sweep to make all of these consistent.